dex/testing/dcr/harness.sh

```diff
@@ -156,6 +166,16 @@ tmux new-window -t $SESSION:1 -n 'alpha'
 tmux send-keys -t $SESSION:1 "set +o history" C-m
 tmux send-keys -t $SESSION:1 "cd ${NODES_ROOT}/alpha" C-m
 
+MINE=1
+if [ -f ./harnesschain.tar.gz ]; then
+    echo "Seeding blockchain from compressed file"
+    MINE=0
+    mkdir -p "${NODES_ROOT}/alpha/data/simnet"
+    mkdir -p "${NODES_ROOT}/beta/data"
+    tar -xzf ./harnesschain.tar.gz -C ${NODES_ROOT}/alpha/data
+    cp -r ${NODES_ROOT}/alpha/data/simnet ${NODES_ROOT}/beta/data/simnet
+fi
```

Comment:
Decred is now our fastest loading harness.

Reply:
So the contents of `harnesschain.tar.gz` are just the `simnet`, right?  All the stuff in `~/dextest/dcr/alpha` is still generated?